src/simulate.js

```diff
@@ -110,4 +110,3 @@ module.exports.launchServer = launchServer;
 module.exports.closeServer = closeServer;
 module.exports.dirs = dirs;
 module.exports.app = server.app;
-module.exports.log = log;
```

Comment:
Is removing the '.log' export intentional? We should probably keep it for backward compatibility with consumers.